smart_reports/html_generator/html_property_cards.py

```diff
@@ -101,7 +110,7 @@ def generate_property_cards(list_top_n_sites: Dict[str, Any] = None) -> str:
               </div>
               <div class="score-item">
                 <div class="value">{competition_score_display}</div>
-                <div class="label">Competition<br />({competing_pharmacy_display} pharmacy)</div>
+                <div class="label">Competition<br />({competing_target_business_display} pharmacy)</div>
```

Comment:
regarding task 3 which i know is not part of this PR , but what i want is to make the hardcoded reference for pharmacy to become f string that we can change

- add a key in the request object such that the complementary business in the html be from genericly saying complementary to say "healthcare", because the user wants to see this name instead of complementary, add another key to say from target expansion business to say "pharmacy" or "صيدلية"


```suggestion
                <div class="label">Competition<br />({competing_target_business_display} {target_business})</div>
```